source-code/website/src/pages/index/Header.tsx

```diff
@@ -1,189 +1,108 @@
 import { createSignal, For, Show } from "solid-js";
+import IconClose from "~icons/material-symbols/close-rounded";
+import IconMenu from "~icons/material-symbols/menu-rounded";
+import IconTwitter from "~icons/cib/twitter";
+import IconGithub from "~icons/cib/github";
 
 export function Header() {
 	const links = [
 		{ name: "Docs", href: "/documentation" },
 		{ name: "Editor", href: "/editor" },
 	];
 
-	const [show, setShow] = createSignal(false);
+	const socialMediaLinks = [
+		{
+			name: "Twitter",
+			href: "https://twitter.com/inlangHQ",
+			Icon: IconTwitter,
+		},
+		{
+			name: "Github",
+			href: "https://github.com/inlang/inlang",
+			Icon: IconGithub,
+		},
+	];
+
+	const [mobileMenuIsOpen, setMobileMenuIsOpen] = createSignal(false);
```

Comment:
mobileMenuIsOpen reads much better. 

Generally, add `is` to the variable name if you deal with boolean values